docs/user/production-considerations/task-manager-troubleshooting.asciidoc

```diff
@@ -569,6 +607,82 @@ Evaluating the preceding health stats, you see the following output under `stats
+
+Evaluating the preceding health stats, you see the following output under `stats.runtime.value.execution.persistence`:
+
+[source,json]
+--------------------------------------------------
+{
+  "recurring": 88, # <1>
+  "non_recurring": 4, # <2>
+  "ephemeral": 8 # <3>
+},
+--------------------------------------------------
+<1> 88% of executed tasks are recurring tasks
+<2> 4% of executed tasks are non-recurring tasks
+<3> 8% of executed tasks are ephemeral tasks
+
+You can infer from these stats that the majority of executions consist of recurring tasks at 88%.
+You can use the `execution.persistence` stats to evaluate the ratio of consumed capacity, but on their own, you should not make assumptions about the sufficency of the availabe capacity.
+
+In order to assess the capacity, you should evaluate these stats against the `load` under `stats.runtime.value`:
+
+[source,json]
+--------------------------------------------------
+{
+    "load": { # <2>
+        "p50": 40,
+        "p90": 40,
+        "p95": 60,
+        "p99": 80
+    }
+}
+--------------------------------------------------
+
+You can infer from these stats that it is very unusual for Task Manager to run out of capacity, so the capacity is likely sufficient to handle the amount of non-recurring and ephemeral tasks.
+
+Suppose an alternate scenario, where you see the following output under `stats.runtime.value.execution.persistence`:
+
+[source,json]
+--------------------------------------------------
+{
+  "recurring": 60, # <1>
+  "non_recurring": 30, # <2>
+  "ephemeral": 10 # <3>
+},
+--------------------------------------------------
+<1> 60% of executed tasks are recurring tasks
+<2> 30% of executed tasks are non-recurring tasks
+<3> 10% of executed tasks are ephemeral tasks
+
+You can infer from these stats that even though most executions are recurring tasks, a substantial percentage of executions are non-recurring and ephemeral tasks at 40%.
+
+Evaluate the `load` under `stats.runtime.value`, we see the following:
+
+[source,json]
+--------------------------------------------------
+{
+    "load": { # <2>
+        "p50": 70,
+        "p90": 100,
+        "p95": 100,
+        "p99": 100
+    }
+}
+--------------------------------------------------
+
+You can infer from these stats that it is quite common for this {kib} instance to run out of capacity.
+Given the high rate of non-recurring and ephemeral tasks, it would be reasonable to assess that there is insufficient capacity in the {kib} cluster to handle the amount of tasks.
+
+It is important to keep in mind that these stats give you a glimpse at a moment in time, and even though there has been insufficient capacity in recent minutes, this might not be true in other times where fewer non-recurring or ephemeral tasks are used. We recommend tracking these stats over time and identifying the source of these tasks before making sweeping changes to your infrastruture.
```

Comment:
```suggestion
It is important to keep in mind that these stats give you a glimpse at a moment in time, and even though there has been insufficient capacity in recent minutes, this might not be true in other times where fewer non-recurring or ephemeral tasks are used. We recommend tracking these stats over time and identifying the source of these tasks before making sweeping changes to your infrastructure.
```